lib/icedfrisby.js

```diff
@@ -1225,31 +1235,54 @@ class Frisby {
 
 
   _makeRequest (done) {
```

Comment:
This is completely rewritten. Easier to compare in split view. Factored out `maybePerformRetry` to avoid duplication.